src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -188,7 +190,10 @@ export class AssetsTable extends React.Component {
   }
 
   render() {
-    return (!this.props.assetsList.length) ? (
+    // TODO: Add UI for when there is nothing in the list and we have a status returned from the API
+    // TODO: http://fhtwd0.axshare.com/#g=1&p=files-and-uploads-empty
```

Comment:
Added these as a reminder to see if we have a ticket created already in the backlog to handle this view.  I talked to Shelby briefly and I don't believe this is needed for feature parity from that discussion, but I didn't' want to forget about it.